core/src/main/java/org/hisp/dhis/android/core/programstageworkinglist/internal/ProgramStageWorkingListAttributeValueFilterStore.kt

```diff
@@ -32,4 +32,8 @@ import org.hisp.dhis.android.core.arch.db.stores.internal.ObjectWithoutUidStore
 import org.hisp.dhis.android.core.programstageworkinglist.ProgramStageWorkingListAttributeValueFilter
 
 internal interface ProgramStageWorkingListAttributeValueFilterStore :
-    ObjectWithoutUidStore<ProgramStageWorkingListAttributeValueFilter>
+    ObjectWithoutUidStore<ProgramStageWorkingListAttributeValueFilter> {
+    fun getProgramStageWorkingListAttributeValueFilterForProgramStageWorkingList(
```

Comment:
As we are in the context of a `ProgramStageWorkingListAttributeValueFilter` store, maybe we could simplify the name of the function to `getForProgramStageWorkingList`.

And the same could apply to the comment above about the legends, maybe it is not needed to repeat the store element name.